src/unity/python/turicreate/toolkits/image_classifier/_annotate.py

```diff
@@ -0,0 +1,216 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2017 Apple Inc. All rights reserved.
+#
+# Use of this source code is governed by a BSD-3-clause license that can
+# be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+"""
+Class definition and utilities for the annotation utility of the image classification toolkit
+"""
+from __future__ import print_function as _
+from __future__ import division as _
+from __future__ import absolute_import as _
+
+from ...visualization import _get_client_app_path
+
+import turicreate as __tc
+
+def _warning_annotations():
+    print(
+        """
+        *** WARNING: If you don't assign the output of the annotate function
+                     to a variable all annotations are going to be lost.
+                     To recover the last annotated SFrame call:
+
+        `tc.image_classifier._recover_annotation()`
```

Comment:
Since this is API we intend for users to call, it shouldn't start with an underscore.

Reply:
Fair point. I always thought of this as an edge-case, something of a recovery state. So kept it hidden. When the user tab completes `tc.image_classifier` recover_annotation doesn't show up. But I see your point. It could be something the user wants to discover.

Reply:
Not only discover - we're telling the user to call it. I think of underscore as more of a marker denoting a public or private/internal method, rather than just as a way of making things hidden from tab completion. (There are other ways to hide things from tab completion if you really want to do just that.)

Reply:
Yes, I would argue that the primary purpose of underscore is tell people that it's an implementation detail and that they _shouldn't_ call it themselves directly